In-house patch to have the rabbitmq providers look in /usr/bin for
rabbitmqadmin instead of /usr/local/bin

--- puppetlabs-rabbitmq-5.3.1/lib/puppet/provider/rabbitmq_binding/rabbitmqadmin.rb.orig	2016-04-21 06:32:55.385984680 -0700
+++ puppetlabs-rabbitmq-5.3.1/lib/puppet/provider/rabbitmq_binding/rabbitmqadmin.rb	2016-04-21 06:37:14.452024943 -0700
@@ -1,15 +1,19 @@
+#######################################################################
+# Oracle has modified the originally distributed contents of this file.
+#######################################################################
+
 require 'json'
 require 'puppet'
 Puppet::Type.type(:rabbitmq_binding).provide(:rabbitmqadmin) do
 
   if Puppet::PUPPETVERSION.to_f < 3
     commands :rabbitmqctl   => 'rabbitmqctl'
-    commands :rabbitmqadmin => '/usr/local/bin/rabbitmqadmin'
+    commands :rabbitmqadmin => '/usr/bin/rabbitmqadmin'
   else
     has_command(:rabbitmqctl, 'rabbitmqctl') do
       environment :HOME => "/tmp"
     end
-    has_command(:rabbitmqadmin, '/usr/local/bin/rabbitmqadmin') do
+    has_command(:rabbitmqadmin, '/usr/bin/rabbitmqadmin') do
       environment :HOME => "/tmp"
     end
   end

--- puppetlabs-rabbitmq-5.3.1/lib/puppet/provider/rabbitmq_exchange/rabbitmqadmin.rb.orig	2016-04-21 06:33:00.401594316 -0700
+++ puppetlabs-rabbitmq-5.3.1/lib/puppet/provider/rabbitmq_exchange/rabbitmqadmin.rb	2016-04-21 06:37:20.410206326 -0700
@@ -1,15 +1,19 @@
+#######################################################################
+# Oracle has modified the originally distributed contents of this file.
+#######################################################################
+
 require 'puppet'
 require File.expand_path(File.join(File.dirname(__FILE__), '..', 'rabbitmqctl'))
 Puppet::Type.type(:rabbitmq_exchange).provide(:rabbitmqadmin, :parent => Puppet::Provider::Rabbitmqctl) do
 
   if Puppet::PUPPETVERSION.to_f < 3
     commands :rabbitmqctl   => 'rabbitmqctl'
-    commands :rabbitmqadmin => '/usr/local/bin/rabbitmqadmin'
+    commands :rabbitmqadmin => '/usr/bin/rabbitmqadmin'
   else
     has_command(:rabbitmqctl, 'rabbitmqctl') do
       environment :HOME => "/tmp"
     end
-    has_command(:rabbitmqadmin, '/usr/local/bin/rabbitmqadmin') do
+    has_command(:rabbitmqadmin, '/usr/bin/rabbitmqadmin') do
       environment :HOME => "/tmp"
     end
   end

--- puppetlabs-rabbitmq-5.3.1/lib/puppet/provider/rabbitmq_queue/rabbitmqadmin.rb.orig	2016-04-21 06:33:06.190244636 -0700
+++ puppetlabs-rabbitmq-5.3.1/lib/puppet/provider/rabbitmq_queue/rabbitmqadmin.rb	2016-04-21 06:37:27.198708092 -0700
@@ -1,15 +1,19 @@
+#######################################################################
+# Oracle has modified the originally distributed contents of this file.
+#######################################################################
+
 require 'json'
 require 'puppet'
 Puppet::Type.type(:rabbitmq_queue).provide(:rabbitmqadmin) do
 
   if Puppet::PUPPETVERSION.to_f < 3
     commands :rabbitmqctl   => 'rabbitmqctl'
-    commands :rabbitmqadmin => '/usr/local/bin/rabbitmqadmin'
+    commands :rabbitmqadmin => '/usr/bin/rabbitmqadmin'
   else
     has_command(:rabbitmqctl, 'rabbitmqctl') do
       environment :HOME => "/tmp"
     end
-    has_command(:rabbitmqadmin, '/usr/local/bin/rabbitmqadmin') do
+    has_command(:rabbitmqadmin, '/usr/bin/rabbitmqadmin') do
       environment :HOME => "/tmp"
     end
   end
